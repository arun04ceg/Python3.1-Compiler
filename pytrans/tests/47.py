y = 10

def f(x):
  global y
  y = 30

print(y)

f(30)

print(y)


