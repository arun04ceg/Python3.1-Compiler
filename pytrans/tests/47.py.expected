(program
 (define f (void))
 (define y (void))
 (set-global! y 10)
 (set-global!
  f
  (lambda (x) (call/ec (lambda (return) (let () (void) (set-global! y 30))))))
 (py-print (get-global y))
 ((get-global f) 30)
 (py-print (get-global y)))
